jclouds-plugin/src/main/resources/jenkins/plugins/jclouds/compute/JCloudsSlaveTemplate/config.jelly

```diff
@@ -96,8 +96,8 @@
           <f:textarea />
         </f:entry>
 
-        <f:entry title="Jenkins User" field="jenkinsUser">
-          <f:textbox />
+        <f:entry title="${%Jenkins Credentials}" field="credentialsId">
```

Comment:
Good question, next one :-)

Seriously: I was about to ask you the same. I never used it myself.

-Fritz

On 02/09/2015 06:52 PM, Marat Mavlyutov wrote:
> In
> jclouds-plugin/src/main/resources/jenkins/plugins/jclouds/compute/JCloudsSlaveTemplate/config.jelly
> <https://github.com/jenkinsci/jclouds-plugin/pull/81#discussion_r24348655>:
> 
>> @@ -96,8 +96,8 @@
>>            <f:textarea />
>>          </f:entry>
>>  
>> -        <f:entry title="Jenkins User" field="jenkinsUser">
>> -          <f:textbox />
>> +        <f:entry title="${%Jenkins Credentials}" field="credentialsId">
> 
> ehmm.. why do we need admin user/password if we use jenkins_user?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/jclouds-plugin/pull/81/files#r24348655>.
>